src/fmt.cc

```diff
@@ -73,6 +73,10 @@ export module fmt;
   export {
 // All library-provided declarations and definitions must be in the module
 // purview to be exported.
```

Comment:
It looks like this comment applies to `FMT_ATTACH_TO_GLOBAL_MODULE` which is not right. Should it be moved some place else?